static/js/apps/eval_rig/util.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Replace [__DC__#(id)(text)] to just text with css class for highlighting.
+ * @param text
+ * @returns
+ */
+export const processText = (text: string): string => {
+  return text.replace(
+    /\[\s*__DC__#(\d+)\(((?:[^)(]+|\([^)]*\))*)\)\s*\]/g,
```

Comment:
Add a comment to explain what's going on?

Reply:
Done